src/walletProvider.ts

```diff
@@ -92,6 +92,22 @@ export class WalletProvider {
         return redirectUrl;
     }
 
+    getMessageSignatureFromWalletUrl(): string {
+        const url = window.location.search.slice(1);
+        console.info("getMessageSignatureFromWalletUrl(), url:", url);
```

Comment:
I assume this console log was used for testing and should be removed now?

Reply:
Kept for debugging (were useful in the past).